### ISSUE TITLE
Microsoft broke Win10XPE again

### ISSUE THREAD

**Issue by ffrags** [ChrisRfr/Win10XPE#14]
Hi ChrisR,

Firstly, thanks for your great work on Win10XPE.  I've been using it for almost 3 years and it has worked great for me and I find it to be a most useful tool.  Thank you so very much.

I have been using your Win10XPE_2021-10-27 release with Windows 11 22000.194 as the source for over a month now and it also has been working flawlessly for me.

Today, however, I tried to refresh some apps in my build and had to download the source from TechBench again.  I guess TechBench published an updated version of the Windows 11 source because in my haste I downloaded Windows 11 22000.318 instead of 22000.194 which is still available there.

Anyway, the build processed successfully using 22000.318 but upon booting up my Win10XPE flash drive I received the error **"Sign-in process initialization failure.  Interactive sign-in process initialization has failed."**  

I then went back to rebuild my project and noticed that I used 22000.318 so I downloaded from TechBench 22000.194 and all worked fine again.

According to Microsoft, Windows 11 is scheduled to receive a major update annually as opposed to Windows 10 which was getting major updates every 6 months.  Still whatever smaller changes they are making and that TechBench is publishing seems to be breaking your very useful tool.

From other forums I know you are very busy on other projects of interest to you,  so I bring this issue up simply to make you aware of it.

Regards,
ffrags

**Comment by ffrags** [comment]
Additional issue, when using Win10 as source for build WMIC.exe works as expected but when using Win11 as source WMIC.exe fails with message "General failure error".  I think Microsoft is making changes to WMIC before they deprecate it completely.

**Comment by ChrisRfr** [comment]
Hello ffrags,
Late reply, a bit disillusioned, PE is just from time to time.

It's OK now for Windows 11 22000.318 update, after a quick look at Re-Win10XPE from Slore.
The wmi test looks good in this WinRE updated version.

In Additions Script, use the Edit button "Add Windows 10 files and folders", add the 2 files below and close (save) notepad.
Check the box to enable this feature during building.
```
; Fix DWM for Windows 11 update (>= 22000.282)
\Windows\System32\wuceffects.dll
; Support to search files in This PC
\Windows\System32\EhStorShell.dll
```

![Alt text](https://i.ibb.co/pj8T5HG/Win11-XPE-22000-318-Wmi.png "Win11XPE_22000.318_Wmi")<br/><br/>

**Comment by ffrags** [comment]
Hello ChrisR,

That fixed it.  Thank you so very much.  Your being available even time to time is better than many others full time.  I just wish I had your knowledge.  Your Win10XPE project is awesome in my humble opinion.  Thank you and wishing you a wonderful new year.
